.devcontainer/Dockerfile

```diff
@@ -1,17 +1,42 @@
 # This is the Dockerfile used to build the image hosted at:
-# ghcr.io/weaversa/cryptol-course:2.13
+# ghcr.io/weaversa/cryptol-course:$CRYPTOL_COURSE_VERSION
 # To use this Dockerfile directly, uncomment the appropriate line in
 # devcontainer.json
 
-FROM ubuntu:22.04
+ARG CRYPTOL_COURSE_TAG=nightly
+
+ARG UBUNTU_VERSION=22.04
+
+ARG CRYPTOL_TAG=nightly
+ARG SAW_TAG=nightly
+ARG WHAT4_SOLVERS_SNAPSHOT=snapshot-20230711
+ARG WHAT4_SOLVERS_ARCHIVE=ubuntu-$UBUNTU_VERSION-X64-bin.zip
+
+ARG CRYPTOL_SERVER_SCHEME=http
+ARG CRYPTOL_SERVER_HOST=0.0.0.0
+ARG CRYPTOL_SERVER_PORT=36681
```

Comment:
@WeeknightMVP The connection problem may be due to different ports being used - here (`36681`) and the one in the workflows (`8080`).